src/PSRule.Rules.Azure/rules/Azure.ACR.Rule.ps1

```diff
@@ -48,4 +48,25 @@ Rule 'Azure.ACR.GeoReplica' -Ref 'AZR-000004' -Type 'Microsoft.ContainerRegistry
     return $Assert.Fail($LocalizedData.ReplicaNotFound);
 }
 
+# Synopsis: Azure Container Registries should have soft delete policy enabled.
+Rule 'Azure.ACR.SoftDelete' -Ref 'AZR-000310' -Type 'Microsoft.ContainerRegistry/registries' -If { GetPreviewLimitations } -Tag @{ release = 'Preview'; ruleSet = '2022_09'; } {
```

Comment:
Let's make this a little more specific so it has less chance of clashing accidentally with a function created in the future.

```suggestion
Rule 'Azure.ACR.SoftDelete' -Ref 'AZR-000310' -Type 'Microsoft.ContainerRegistry/registries' -If { GetACRSoftDeletePreviewLimitations } -Tag @{ release = 'Preview'; ruleSet = '2022_09'; } {
```

Reply:
Agreed, this makes sense!